### PR TITLE
overhauled the custom URL page

### PR DESCRIPTION
**Pull request by wslack**
cc @gemfarmer and @jmhooper

### CODE REVIEW

File: pages/using-federalist/launch-checklist.md [hunk 1]
```diff
@@ -9,6 +9,6 @@ We've worked to make launching a live site on Federalist as simple as possible.
 
 1. Sign an IAA with 18F to use Federalist.
 2. Verify that your agency accepts the risk of launching a site on Federalist.
-3. Build your site and verify that the site is ready for launch.
-4. Initiate setup of your custom URL within your agency and with the Federalist team. Federalist will set up support for your URL and provide a subdomain to which your technical staff will delegate their DNS.
-5. Alter your site's settings in the Federalist web interface to have Federalist manage your site's assets correctly during build.
+3. Build your site and verify that the site is ready for launch using our preview URLs.
+4. Initiate setup of your custom URL within your agency and with the Federalist team. At your command, Federalist will initiate the process and provide you with a CloudFront URL such as https://d2xyasfn4889hb.cloudfront.net/. You will then configure DNS records with a CNAME to the CloudFront URL. Once this is complete, your site content will be available at your agency URL but with broken images. Technical background on this process is [here](/pages/how-federalist-works/custom-urls/).
```
Comment (jmhooper) [inline pages/using-federalist/launch-checklist.md]: How long does it usually take for us to get the :+1: from GSA IT? If it's a significant amount of time, we may want to say something about that so people don't mistake that it's just a switch we flip on.

Comment (wslack) [inline pages/using-federalist/launch-checklist.md]: oh good call; we should add a piece on that. It's just 24 hours but it is a thing.

**Comment by wslack** [comment]
updated!

**Comment by jmhooper** [comment]
:+1:

**Pull request merged**